igor-web/src/main/java/com/netflix/spinnaker/igor/scm/gitlab/client/model/Commit.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 bol.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm.gitlab.client.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Date;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Commit {
+    public final String id;
+    public final String author_name;
+    public final Date authored_date;
+    public final String message;
+
+    @JsonCreator
+    public Commit(@JsonProperty("id") String id,
```

Comment:
You can remove the annotation value for `id` and `message`, as Jackson will default to the field name. Leave it for `author_name` and `authored_date` (if you rename the fields, that is).

Reply:
Jackson gets confused if I remove them.